src/video_core/texture_cache/texture_cache.h

```diff
@@ -1492,4 +1619,30 @@ bool TextureCache<P>::IsFullClear(ImageViewId id) {
            scissor.max_y >= size.height;
 }
 
+template <class P>
+void TextureCache<P>::UpdateFramebufferReferences() {
+    for (u32 i = 0; i < slot_images.Size(); ++i) {
+        const SlotId id{i};
+        if (slot_images.IsIndexFree(id)) {
```

Comment:
```suggestion
        if (!slot_images.IsIndexFree(id)) {
            continue;
        }
```

Would allow unindenting most of the code a little more.